### PR TITLE
adding a 5th kind

### PR DESCRIPTION
**Pull request by bsandipan88**
Added a 5th Kind

### CODE REVIEW

**Comment by Cooksey99** [comment]
There isn't an issue with it, but what's the purpose of this? What's the significance of this addition?

File: README.md [hunk 1]
```diff
@@ -34,24 +35,25 @@ Lets make a concrete example:
  PO moves this to TODO.
 
  A developer PULLS this task, means: moves this task from TODO into IN PROGRESS>
- Once she is done, set the task into "COMPLETED the current step". So small detail:
+ Once He/She is done, set the task into "COMPLETED the current step". So small detail:
```
Comment (virgiliolino) [inline README.md:38]: She/he at least please :)

**Review by virgiliolino — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (bsandipan88) [reply README.md:38]: Made the changes.

Comment (Cooksey99) [reply README.md:38]: I can squash and merge this, but we need to fix the check that's failing. Let me fix that and merge.

**Review by Cooksey99 — APPROVED** [review]
It sure reads better, now. Thanks!

**Pull request merged**